models/spring-ai-elevenlabs/src/main/java/org/springframework/ai/elevenlabs/ElevenLabsTextToSpeechOptions.java

```diff
@@ -0,0 +1,410 @@
+/*
+ * Copyright 2025-2025 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.ai.elevenlabs;
+
+import java.util.List;
+import java.util.Objects;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import org.springframework.ai.elevenlabs.api.ElevenLabsApi;
+import org.springframework.ai.audio.tts.TextToSpeechOptions;
+
+/**
+ * Options for ElevenLabs text-to-speech.
+ *
+ * @author Alexandros Pappas
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class ElevenLabsTextToSpeechOptions implements TextToSpeechOptions {
+
+	@JsonProperty("model_id")
+	private String modelId;
+
+	// Path Params
+	@JsonProperty("voice_id")
+	private String voiceId;
+
+	// End Path Params
+
+	// Query Params
+	@JsonProperty("enable_logging")
+	private Boolean enableLogging;
+
+	@JsonProperty("output_format")
+	private String outputFormat;
+
+	// End Query Params
+
+	@JsonProperty("voice_settings")
+	private ElevenLabsApi.SpeechRequest.VoiceSettings voiceSettings;
+
+	@JsonProperty("language_code")
+	private String languageCode;
+
+	@JsonProperty("pronunciation_dictionary_locators")
+	private List<ElevenLabsApi.SpeechRequest.PronunciationDictionaryLocator> pronunciationDictionaryLocators;
+
+	@JsonProperty("seed")
+	private Integer seed;
+
+	@JsonProperty("previous_text")
+	private String previousText;
+
+	@JsonProperty("next_text")
+	private String nextText;
+
+	@JsonProperty("previous_request_ids")
+	private List<String> previousRequestIds;
+
+	@JsonProperty("next_request_ids")
+	private List<String> nextRequestIds;
+
+	@JsonProperty("apply_text_normalization")
+	private ElevenLabsApi.SpeechRequest.TextNormalizationMode applyTextNormalization;
+
+	@JsonProperty("apply_language_text_normalization")
+	private Boolean applyLanguageTextNormalization;
+
+	public static Builder builder() {
+		return new ElevenLabsTextToSpeechOptions.Builder();
+	}
+
+	@Override
+	@JsonIgnore
+	public String getModel() {
+		return getModelId();
+	}
+
+	@JsonIgnore
+	public void setModel(String model) {
+		setModelId(model);
+	}
+
+	public String getModelId() {
```

Comment:
do we need these getter and setters?  why not just normalize usage to getModel and set the json serialization value to be 'modelId'.   The same can be said for the property getVoice and 'voiceId'